pants.ini

```diff
@@ -176,6 +166,15 @@ no_warning_args: [
     '-S-nowarn',
   ]
 
+[lint.checkstyle]
```

Comment:
Yeah, I'll kill the migrate_config script, since we no longer use it...

@stuhood your thoughts re my reply to you above, re a `check` goal?